site/src/main/paradox/Apache-Beam.md

```diff
@@ -29,6 +29,7 @@ Also check out the [SDK Version Support Status](https://cloud.google.com/dataflo
 
 | **Scio Version** | **Beam Version** | **Details** |
 |:----------------:|:----------------:|:------------|
+| 0.10.4+          | 2.30.0           | This version will be deprecated on [TBD](https://cloud.google.com/dataflow/docs/support/sdk-version-support-status). |
 | 0.10.3+          | 2.29.0           | This version will be deprecated on April 29, 2022. |
```

Comment:
Since 2.29.0 only applies to 0.10.3 after the new release.
```suggestion
| 0.10.3           | 2.29.0           | This version will be deprecated on April 29, 2022. |
```